src/app/grid-auto-size/grid-auto-size.sample.ts

```diff
@@ -1,8 +1,8 @@
-import { Component, ViewChild, OnInit } from '@angular/core';
+import { Component, ViewChild, OnInit, HostBinding } from '@angular/core';
 import { NgFor } from '@angular/common';
 
 import { SAMPLE_DATA } from '../shared/sample-data';
-import { DisplayDensity, GridSelectionMode, IgxGridComponent, IGX_BUTTON_GROUP_DIRECTIVES, IGX_GRID_DIRECTIVES } from 'igniteui-angular';
+import { GridSelectionMode, IgxGridComponent, IGX_BUTTON_GROUP_DIRECTIVES, IGX_GRID_DIRECTIVES, Size } from 'igniteui-angular';
```

Comment:
Oh, do we really intend to still keep exposing an enum for this? I sort of expected that to be internal, so it would have no place in any sample. Below sizes can be typed with an union like `'small' | 'medium' | 'large'` just fine